default.nix

```diff
@@ -4,15 +4,24 @@ let
         buildGoModule
         statik
         git
+        python3
         runCommand;
 in
 let
     src = pkgs.copyPathToStore ./.;
+    versionFile = builtins.fetchurl https://api.github.com/repos/replit/upm/commits/master;
```

Comment:
This will always pull the latest SHA right? This may give us incorrect version tagging if upm/master is ahead of the commit we have pinned in goval.

Maybe we can make version be an argument and then we can pass it in when we import the package in the pid1 derivation (since the fetchgit result should contain the SHA)

Reply:
Yeah, that could work too. That way we get the best of both worlds, though I wonder if we even need a dynamic version if we're always going to fetch the pinned sha from github :thinking:

Reply:
Indeed, the main reason we want any of this is to know what version of the binaries we have in prod. As long as we know the git SHA for pid1, that lets us trace all the versions of its dependencies since we pin them via niv.

Reply:
I can switch to a parameter based versionName in the bash if for now, dynamic versioning would be nice to have if we ever want to build and publish the derivations from CI.